share/functions/__fish_complete_users.fish

```diff
@@ -2,11 +2,11 @@
 # completion. If you just want a list of user names use __fish_print_users.
 function __fish_complete_users --description "Print a list of local users, with the real user name as a description"
     if command -sq getent
-        command getent passwd | cut -d : -f 1,5 | string replace -r ':' \t
+        command getent passwd | cut -d : -f 1,5 | string replace -r ':' \t | cut -d',' -f1
```

Comment:
We're trying to cut down on usage of `cut` since it requires forking off an external tool.

The first `cut` isn't easily replaced with `string` (which is why it is still there), but this one is - `string replace -r ',.*' ''`.

Also, a comment would be nice.